app/Http/Controllers/DataResourceController.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Lintol\Capstone\Models\DataResource;
+use Illuminate\Http\Request;
+use Lintol\Capstone\Transformers\DataResourceTransformer;
+
+class DataResourceController extends Controller
+{
+
+    public function __construct(DataResourceTransformer $transformer)
+    {
+        $this->transformer = $transformer;
+    }
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        //
+        $data = DataResource::all();
+
+        return fractal()
+            ->collection($data, $this->transformer, 'dataResources')
+            ->respond();
+    }
+
+    /**
+     * Show the form for creating a new resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function create()
+    {
+        //
```

Comment:
Just to avoid confusion, and for for consistency with the other controllers, I'd maybe suggest leaving out the inactive methods - create, show, etc.?